coderd/coderdtest/coderdtest.go

```diff
@@ -311,6 +314,10 @@ func NewOptions(t testing.TB, options *Options) (func(http.Handler), context.Can
 		options.NotificationsEnqueuer = &testutil.FakeNotificationsEnqueuer{}
 	}
 
+	if options.OneTimePasscodeValidityPeriod == 0 {
+		options.OneTimePasscodeValidityPeriod = 20 * time.Second
```

Comment:
This may be too short for tests since they're running in parallel. I would recommend setting it explicitly in those tests that require a shorter period. Using `testutil.WaitLong` may be slightly safer, but we don't really know anything about how the goroutines will be scheduled during tests.